exasol_script_languages_container_ci_setup/lib/aws/wrapper/datamodels/common.py

```diff
@@ -0,0 +1,12 @@
+import dataclasses
+from typing import Any
+
+
+@dataclasses.dataclass(frozen=True)
+class ARN:
+    aws_arn: Any
+
+
+@dataclasses.dataclass(frozen=True)
```

Comment:
What is the point of wrapping it in a `dataclass`?

Reply:
ah, explicit typing and making us independent of the actual type we get from the AWS library

Reply:
See comment above ([NewType](https://docs.python.org/3.8/library/typing.html#newtype))